src/table/Exposure.cc

```diff
@@ -186,61 +186,72 @@ void ExposureFitsWriter::_writeRecord(BaseRecord const & r) {
 //----- ExposureFitsReader ---------------------------------------------------------------------------------
```

Comment:
That's actually intentional - I'm trying to emphasize that the following class (`PersistableObjectColumnReader`) is just a helper class for (and implementation detail of) `ExposureFitsReader`.